source/mainnet/net/guides/export-import.rst

```diff
@@ -5,20 +5,50 @@
 Make a backup of identities, accounts, and addresses
 ====================================================
 
-To make sure that you have a backup of your accounts, identities, and addresses, Concordium strongly recommends that you export the data to a file you can store in a safe location. The backup will ensure that you can recover your accounts, identities, and addresses if your Wallet database becomes damaged or if, for some reason, you can't access the Wallet.
-
 .. Note::
 
-    Backup and import cannot be used for |mw-gen2| because it uses a secret recovery phrase to recover the wallet. For more information, see :ref:`Recover your wallet<recover-wallet>`.
+   This information is not relevant for |mw-gen2| or |bw| which use a secret recovery phrase to recover a wallet. For more information, see :ref:`Recover your wallet<recover-wallet>`.
 
-.. Warning::
-    You can't import a file created in the |mw-gen1| into the Desktop Wallet or the other way around because the two wallets handle private keys in different ways. You also cannot import a backup from the |mw-gen1| to the |mw-gen2|. If you try to import a file that has been exported from the |mw-gen1| into the Desktop Wallet, the import will fail, and likewise, if you try to import a file exported from the Desktop Wallet into the |mw-gen1|. For more information, see :ref:`Deciding between the Wallets <choosing-wallet>`.
+To make sure that you have a backup of your accounts, identities, and addresses, Concordium strongly recommends that you export the data to a file you can store in a safe location. The backup will ensure that you can recover your accounts, identities, and addresses if your Wallet database becomes damaged or if, for some reason, you can't access the Wallet.
```

Comment:
It is not possible to make a backup of identities. Nor of accounts for that matter in the new wallets. So I am a bit confused about this recommendation. Is it in some context that I don't see?